docs/cloudformation.json

```diff
@@ -870,7 +870,10 @@
     },
 
     "CustomResourcesQueue": {
-      "Type": "AWS::SQS::Queue"
+      "Type": "AWS::SQS::Queue",
+      "Properties": {
+        "VisibilityTimeout": "1800"
```

Comment:
This seems really long - you handle de-duping requests here as well, right?  Otherwise we could see things being re-submitted 30 minutes later after a crash or something.  In the case where de-duping *is* handled, what's the idea behind making this so long?  Would 5 minutes be sufficient?